sdk/loadtestservice/azure-mgmt-loadtestservice/samples/creating_load_test_resource.py

```diff
@@ -0,0 +1,63 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: creating_load_test_resource.py
```

Comment:
I believe mgmt plane samples should be in this folder https://github.com/Azure-Samples/azure-samples-python-management/tree/main/samples cc @msyyc

Reply:
CC @ninallam

Reply:
@NiveditJain Please move this testcase to https://github.com/Azure-Samples/azure-samples-python-management/tree/main/samples since it is for mgmt package.

Reply:
Done @msyyc!